### ISSUE TITLE
[Bug?]: Yarn playground does not work due to CORS issues

### ISSUE THREAD

**Issue by wojtekmaj** [yarnpkg/berry#4828]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

I was trying to use Yarn playground to reproduce issue as instructed here:

https://github.com/yarnpkg/berry/issues/4711#issuecomment-1236302845

However, upon load, and upon clicking "Run", I'm getting `TypeError: Failed to fetch` errors, preceded by:

> Access to fetch at 'https://viko0.sse.codesandbox.io/api/check-repo' from origin 'https://yarnpkg.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### To reproduce

1. Open http://yarnpkg.com/playground
2. See Console, observe errors

OR

1. Open http://yarnpkg.com/playground
2. Click Run, observe how loader never stops

### Environment

```shell
System:
    OS: macOS 12.4
    CPU: (8) arm64 Apple M1 Pro
  Binaries:
    Node: 16.16.0 - /private/var/folders/7r/rpcp4wz97jngh5x0nk88p5br0000gp/T/xfs-ec4d6ef9/node
    Yarn: 3.2.3 - /private/var/folders/7r/rpcp4wz97jngh5x0nk88p5br0000gp/T/xfs-ec4d6ef9/yarn
    npm: 8.11.0 - /opt/homebrew/bin/npm
```

### Additional context

Accessing https://viko0.sse.codesandbox.io/ directly also does not help.

**Comment by RDIL** [comment]
Duplicate of #4760
